KomaMRIPlots/Project.toml

```diff
@@ -24,6 +25,7 @@ KomaMRIBase = "0.9"
 MAT = "0.10"
 PlotlyJS = "0.18"
 PlutoPlotly = "0.4, 0.5"
+QMRIColors = "1.0.0"
```

Comment:
Just use "1"

Reply:
fixed in commit [479f317](https://github.com/JuliaHealth/KomaMRI.jl/pull/528/commits/479f31791aa5cec0e9af61a9e83d1bf99f680cd2)